polar_plotter_AREA.py

```diff
@@ -0,0 +1,180 @@
+## Loop over files
+# Declare list for each file's gain list
+gains = []
+realized_gains = []
+
+#Define ranges of lines to be read for azimuthal gains at the given freq_num
+#There are 2668 lines for each frequency block (including headers)
+#And azimuthal gain values are lines 4-40 counting from the header of each freq
+#So read lines between (2668*freq_num)+4 and (2668*freq_num)+40
+azimuthGainStart = ((g.freq_num-1)*2668)+4
+azimuthGainEnd = ((g.freq_num-1)*2668)+40
+
+#print(azimuthGainStart)
+#print(azimuthGainEnd)
+
+for i in range(g.initial_antenna, g.final_antenna+1):
+	# Gains
+	## Open the file to read
+	with open(g.source1 + "/" + "child" + "_" + str(i) + ".txt", "r") as f:
+		## Read in the file
+		uan_read = csv.reader(f, delimiter = ' ')
+		## Declare list to hold azimuth gains
+		azimuth_gain = []
+		for m, row in enumerate(uan_read):
+			## skip the header
+                        ## for AREA: counting from frequency header, rows 4-40 for azimuthal gain 
+			#if m >= 4 and m <= 40:
+                        if m >= azimuthGainStart and m <= azimuthGainEnd:
+				## Let's just add the azimuthal angles
+                                print(row[4])
+                                azimuth_gain.append(float(row[4]))
+	f.close()
+	### Declare temporary zenith gains
+	#zenith_gains = []
+	### Loop over the zenith angles
+	#for j in range(1, 36):
+	#	## Declare temporary azimuth means
+	#	azimuth_mean = []
+	#	## Loop over the azimuth angles
+	#	k = j*73
+	#	while k < ((j+1)*73):
+	#		azimuth_mean.append(azimuth_gain[k])
+	#		## Remember to increment!
+	#		k += 1
+	#	zenith_gains.append(np.mean(azimuth_mean))
+		## Remember to increment!
+	## Append the list of mean azimuth gains at each zenith to the gains list
+	gains.append(azimuth_gain)
+
+	# Realized gains
+
+	#with open(g.source2 + "/" + "/" + str(i) + "_" + str(g.freq_num) + ".uan", "r") as f:
+	#	## Read in the file
+	#	uan_read = csv.reader(f, delimiter = ' ')
+	#	## Declare list to hold azimuth gains
+	#	azimuth_gain = []
+	#	for m, row in enumerate(uan_read):
+	#		## skip the header
+	#		if m >= 17:
+	#			## Let's just add the azimuthal angles
+	#			azimuth_gain.append(float(row[2]))
+	#f.close()
+	# Declare temporary zenith gains
+	#zenith_gains = []
+	## Loop over the zenith angles
+	#for j in range(1, 36):
+	#	## Declare temporary azimuth means
+	#	azimuth_mean = []
+	#	## Loop over the azimuth angles
+	#	k = j*73
+	#	while k < ((j+1)*73):
+	#		azimuth_mean.append(azimuth_gain[k])
+	#		## Remember to increment!
+	#		k += 1
+	#	zenith_gains.append(np.mean(azimuth_mean))
+	#	## Remember to increment!
+	## Append the list of mean azimuth gains at each zenith to the gains list
+	#realized_gains.append(zenith_gains)
+
+
+print(len(azimuth_gain))
+print(len(gains))
+#print(len(gains[0]))
+#
+#
+#
+## Plotting
+# Make a list for the zenith angles
+zenith_angles = []
+flipped_zenith_angles = [] ## For making a version with realized gains on the other side
+for i in range(0, 37):
```

Comment:
If the start is `0`, you can omit it:
```suggestion
for i in range(37):
```